### ISSUE TITLE
Tic-tac-toe tutorial template does not work when downloading zip export

### ISSUE THREAD

**Issue by tomtheisen** [reactjs/react.dev#6139]
When I try to run the [tic tac toe tutorial](https://react.dev/learn/tutorial-tic-tac-toe) locally, I follow the instructions.

1. Click the codesandbox "Fork" link.
2. Click the download zip export.
3. Extract to folder.
4. `npm install`
5. `npm start`

This starts a web browser showing an error message. 

```
`'React' must be in scope when using JSX  react/react-in-jsx-scope`
```

I found a [blog entry regarding this](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).  It indicates that babel v7.9 is required, or some other compatible build tool.  I found that I was using `react-scripts@1.0.0`, which specifies a lower version of babel-core.

In the sandbox, `package.json` looks like this, which seems fine.

```json
{
  "dependencies": {
    "react": "^18.0.0",
    "react-dom": "^18.0.0",
    "react-scripts": "^5.0.0"
  },
  "main": "/index.js",
  "devDependencies": {}
}
```

But after downloading the zip, it looks like this.

```json
{
  "dependencies": {
    "react": "^18.0.0",
    "react-dom": "^18.0.0",
    "react-scripts": "^5.0.0"
  },
  "main": "/index.js",
  "devDependencies": {
    "react-scripts": "1.0.0"
  },
  "name": "ljg0t8",
  "description": null,
  "version": "0.0.0",
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject"
  }
}
```

Note the extra `devDependencies` entry.  The codesandbox zip contents do not match the files in the sandbox editor.  It seems the zip file will need to be hosted somewhere more trustworthy.

**Comment by danielmayorga** [comment]
I want to add a bit more to this issue. This discrepancy seems to be an issue with codesandbox/codesandbox-client#7052 defaulting to using a template when there isn't a `src` folder. I haven't looked too much at their source code to figure out the bug.

One workaround if we want to keep the export instructions with codesandbox is to explicitly define the `src` folder and update `package.json` to include the scripts you require to no longer rely on this template behavior for export in codesandbox. One drawback is that the page has a bunch of screenshots from the existing codesandbox instance, so I don't know if this is an approach you'd want to take.

I've forked of the embedded codesandbox [example](https://codesandbox.io/s/723948?file=/App.js&utm_medium=sandpack) on the page to contain a `src` folder and added scripts to package.json to no longer depend on the template. Here is the [fork](https://codesandbox.io/s/upbeat-leftpad-ry8zyc?file=/package.json). You should be able to download the zip and have a consistent `package.json` without the issue seen above.

I have other suggestions or approaches:
- Like Tom has stated, we could store the files in a place where they wont be messed with by templates.
- We can remove the follow locally instruction because right now I think that template behavior is preventing it from running locally.
- We could wait for codesandbox to fix their issue but this isn't a guarantee that it will fix ours because we currently depend on the template behavior to assign `scripts` in `package.json`

**Comment by tomtheisen** [comment]
I believe there should be a template that's able to be followed locally.  Currently, the "Installation" section of the docs is pushing big second-order frameworks pretty hard.  There's [a section about just using react](https://react.dev/learn/add-react-to-an-existing-project#using-react-for-a-part-of-your-existing-page), but it's a bit harder to approach for a beginner.  Beginners are probably the intended audience of the tic tac toe tutorial.

Basically, I believe there should be a straight-forward way to get from nothing to hello world (or tic tac toe).  The sandbox is nice, but personally, I wouldn't want to use it for really developing an understanding.  The sandbox is doing more things than I understand, (such as `import`-ing .css files).  Not everyone is like me, but I prefer to learn one thing at a time, and I think these docs are worse for not having that.

**Comment by lunaleaps** [comment]
Thanks for the report and the workaround @danielmayorga! I'll close this issue to continue the discussion in https://github.com/reactjs/react.dev/issues/6317. 

I'll link to your codesandbox fork as a workaround as well. I want to get the status of the codesandbox issue. I like the approach of adding a `src` folder. That may a good path forward if there doesn't seem to be movement on the codesandbox issue
